### ISSUE TITLE
band selection in construct_kdotp part

### ISSUE THREAD

**Issue by tpyang** [Z2PackDev/TBmodels#114]
In the function MODEL_TB.construct_kdotp(k_star, order=2), the program will construct the kp model with same band number as the given TB model. 

Is it possible to construct a kp model with much less band number, for example 3 bands around Gamma from a multy-band W90 model? 

Thank you very much,
Yang

**Comment by greschd** [comment]
No, TBmodels doesn't contain an implementation for such a disentanglement routine. The current TB -> k.p conversion is just a straightforward Taylor expansion.
